### PR TITLE
More on `EventFactory` and stringifier improvements

### PR DESCRIPTION
**Pull request by alexander-yevsyukov**
This PR:
* continues migration to creating events and event contexts via `EventFactory`;
* moves enrichment utilities into a separate class;
* improves working with quotes in `ListStringifier` and `MapStringifier`;
* bumps Gradle version to 3.5.

#### Obtaining enrichments
After this PR getting enrichments is done like this:
```java
   Optional<Enrichment.Container> ec = Enrichments.getEnrichments(event.getContext());
```

### CODE REVIEW

**Comment by alexander-yevsyukov** [comment]
@IlliaShepilov, please have a look at the regexp issue that I mentioned in the chat.

**Comment by alexander-yevsyukov** [comment]
@armiol, @IlliaShepilov is going to add some changes that we discussed via chat.

@IlliaShepilov, please notify @armiol when you finish.

**Comment by IlliaShepilov** [comment]
@armiol PTAL.

File: build.gradle [hunk 1]
```diff
@@ -33,7 +33,7 @@ buildscript {
         guavaVersion = '20.0'
         protobufGradlePluginVerison = '0.8.0'
         
-        spineVersion = '0.8.31-SNAPSHOT'
+        spineVersion = '0.8.32-SNAPSHOT'
```
Comment (armiol) [inline build.gradle]: Before merge, please take a look at the most recent Spine version and increase this one accordingly..

File: client/src/main/java/org/spine3/base/Enrichments.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        return isEnabled;
+    }
+
+    /**
+     * Returns all enrichments from the context.
+     *
+     * @param context a context to get enrichments from
+     * @return an optional of enrichments
+     */
+    public static Optional<Enrichment.Container> getEnrichments(EventContext context) {
```
Comment (armiol) [inline client/src/main/java/org/spine3/base/Enrichments.java]: Could you please have a code sample in the PR description? That would make any migration so much easier for the existing framework users.

**Pull request merged**